src/main/java/pageobject/NewApplicationFirstFormPage.java

```diff
@@ -0,0 +1,56 @@
+package pageobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class NewApplicationFirstFormPage extends AbstractPage {
+    private static final By TRADE_NAME_FIELD_LOCATOR = By.xpath("//input[@id='input-3']");
```

Comment:
by.id ?

Reply:
same as above, but if you think it is good point for leave comment, ok i'll fix it